src/snark/src/algebra/fields/fp3.tcc

```diff
@@ -155,19 +159,6 @@ Fp3_model<n,modulus> Fp3_model<n,modulus>::sqrt() const
     Fp3_model<n,modulus> x = (*this) * w;
     Fp3_model<n,modulus> b = x * w; // b = (*this)^t
 
-#if DEBUG
-    // check if square with euler's criterion
-    Fp3_model<n,modulus> check = b;
-    for (size_t i = 0; i < v-1; ++i)
-    {
-        check = check.squared();
-    }
-    if (check != one)
-    {
-        assert(0);
-    }
```

Comment:
This should be unconditional, not deleted: https://github.com/syd0/zcash/blob/master/src/snark/src/algebra/fields/fp2.tcc#L161-L170 . Also change ``assert`` to ``assert_except``.